plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -311,6 +311,23 @@
               version="1.0.1" />
       </libraryFile>
     </library>
+    
+    <library
+          id="cloudtranslation"
+          group="clientapis"
+          name="%cloudtranslation.name"
+          tooltip="%cloudtranslation.tooltip"
+          javaVersion="1.8"
+          siteUri="https://cloud.google.com/translate/docs/reference/libraries" >
+      <libraryDependency id="googlecloudcore" />
+      <libraryFile
+            javadocUri="https://googlecloudplatform.github.io/google-cloud-java/0.20.0/apidocs/index.html?com/google/cloud/translate/package-summary.html">    
```

Comment:
Looks like this can be 0.21.0.  I did see on Maven Central that the latest version of these libraries is now 0.21.1 (not 0.22.0 as I thought), though some are 0.21.1-alpha and some are 0.21.1-beta.